toolium/utils/dataset.py

```diff
@@ -226,31 +230,62 @@ def _get_random_phone_number():
 def _replace_param_transform_string(param):
     """
     Transform param value according to the specified prefix.
-    Available transformations: DICT, LIST, INT, FLOAT, STR, UPPER, LOWER
+    Available transformations: DICT, LIST, INT, FLOAT, JSON, STR, UPPER, LOWER, REPLACE, TITLE
 
     :param param: parameter value
     :return: tuple with replaced value and boolean to know if replacement has been done
     """
-    type_mapping_regex = r'\[(DICT|LIST|INT|FLOAT|STR|UPPER|LOWER):(.*)\]'
+    type_mapping_regex = r'\[(DICT|LIST|INT|FLOAT|JSON|STR|UPPER|LOWER|REPLACE|TITLE):([\w\W]*)\]'
     type_mapping_match_group = re.match(type_mapping_regex, param)
     new_param = param
     param_transformed = False
 
     if type_mapping_match_group:
         param_transformed = True
-        if type_mapping_match_group.group(1) == 'STR':
-            new_param = type_mapping_match_group.group(2)
-        elif type_mapping_match_group.group(1) in ['LIST', 'DICT', 'INT', 'FLOAT']:
-            exec('exec_param = {type}({value})'.format(type=type_mapping_match_group.group(1).lower(),
-                                                       value=type_mapping_match_group.group(2)))
-            new_param = locals()['exec_param']
-        elif type_mapping_match_group.group(1) == 'UPPER':
-            new_param = type_mapping_match_group.group(2).upper()
-        elif type_mapping_match_group.group(1) == 'LOWER':
-            new_param = type_mapping_match_group.group(2).lower()
+        if type_mapping_match_group.group(1) in ['DICT', 'LIST', 'INT', 'FLOAT', 'JSON']:
+            if '::' in type_mapping_match_group.group() and 'FLOAT' in type_mapping_match_group.group():
+                params_to_replace = type_mapping_match_group.group(
+                    2).split('::')
+                float_formatted = "{:.2f}".format(round(float(params_to_replace[0]), int(params_to_replace[1])))
+                new_param = float_formatted
+            elif type_mapping_match_group.group(1) == 'JSON':
+                new_param = json.loads(type_mapping_match_group.group(2))
```

Comment:
Aquí metería la lógica comentada más arriba para intentar tanto dict() como json.loads() en el caso de DICT, del tipo:
```suggestion
            elif type_mapping_match_group.group(1) == 'DICT':
                try:
                  json.loads(xxx)
                except json.decoder.JSONDecodeError:
                  dict(xxx)
```